api/Blob.json

```
@@ -87,7 +87,7 @@
             "nodejs": [
               {
                 "version_added": "15.7.0",
-                "notes": "Before v16.7.0, the <code>endings</code> option use a non-standard implement."
+                "notes": "Before v16.7.0, the <code>endings</code> option represents the character encoding to use for string sources, not the line endings in string source parts per the specification."
```

I think this isn't quite right. The History in the Node.js docs states:

> v16.7.0: Added the standard `endings` option to replace line-endings, and removed the non-standard `encoding` option.

This states two things:
1. The `endings` option was only added in v16.7.0.
2. Before v16.7.0, a non-standard `encoding` option was supported (but as it's non-standard I wouldn't mention it).

So I would suggest to put `partial_implementation: true` for this range, mentioning that the `endings` option was not supported:

```suggestion
                "notes": "Before v16.7.0, the <code>endings</code> option was not supported."
```